src/meta-client/src/client.rs

```diff
@@ -264,18 +264,30 @@ impl ClusterInfo for MetaClient {
 
         let mut nodes = if get_metasrv_nodes {
             let last_activity_ts = -1; // Metasrv does not provide this information.
+
+            // TODO(dennis): Get Metasrv node info
```

Comment:
Looks like we can't retrieve the metasrv node info in `meta-client`. @fengjiachun

Reply:
I'm going to add it in next PR.

Reply:
Ok